lib/include/jxl/encode.h

```diff
@@ -361,6 +361,65 @@ JXL_EXPORT JxlEncoderStatus JxlEncoderProcessOutput(JxlEncoder* enc,
                                                     uint8_t** next_out,
                                                     size_t* avail_out);
 
+/**
+ * Sets the frame information for this frame to the encoder. This includes
+ * animation information such as frame duration to store in the frame header.
+ * The frame header fields represent the frame as passed to the encoder, but not
+ * necessarily the exact values as they will be encoded file format: the encoder
+ * could change crop and blending options of a frame for more efficient encoding
+ * or introduce additional internal frames. Animation duration and time code
+ * information is not altered since those are immutable metadata of the frame.
+ *
+ * TODO(lode): JxlFrameHeader currently only contains animation information,
+ * also allow setting crop and blending fields.
```

Comment:
Actually since https://github.com/libjxl/libjxl/pull/831 it does already contain `JxlLayerInfo` which has crop and blend info. So I think this TODO can be dropped (though it still needs to be implemented to actually take that info into account in the encoder). Can remove the TODO later too.

Reply:
Will remove it later :)